### PR TITLE
Add Statically CDN

### PR DESCRIPTION
**Pull request by fransallen**
Statically is a free & public CDN for static files.

It is used by many developers to get their projects online quickly or as a CDN on their technology stack.

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -10739,6 +10739,17 @@
...
     },
+    "Statically": {
+      "cats": [
+        31
+      ],
+      "headers": {
+        "Server": "^statically$"
+      },
+      "icon": "Statically.svg",
+      "html": "<[^>]+?//cdn\\.statically\\.io/",
```
Comment (AliasIO) [inline src/apps.json:10750]: This will identify the CDN even when a website links to a file hosted on it. Is there a better way to identify it?

Comment (wxh06) [reply src/apps.json:10750]: See #3141 as reference.
https://github.com/AliasIO/wappalyzer/blob/b956b4e6075bffa4a2b9ee4d50b84c044d1c6df3/src/apps.json#L13389,L13397

Comment (wxh06) [reply src/apps.json:10750]:
```suggestion
      "html": "<link [^>]*?href=\"?[a-z]*?:?//cdn\\.statically\\.io/",
```

**Comment by fransallen** [comment]
Ok done, can you check?

**Comment by AliasIO** [comment]
Missed your update, sorry about that! Thanks.

**Pull request merged**